docs/core/deploying/deploy-with-cli.md

```diff
@@ -100,44 +104,55 @@ When you run the app ([`dotnet run`](../tools/dotnet-run.md)), the following out
+Your app is configured to target a specific version of .NET. That targeted .NET runtime is required to be on any machine where your app runs. For example, if your app targets .NET Core 3.1, any machine that your app runs on must have the .NET Core 3.1 runtime installed. As stated in the [Publishing basics](#publishing-basics) section, you can edit your project file to change the default target framework or to target more than one framework.
 
-Publishing an FDE creates an app that automatically rolls-forward to the latest .NET Core security patch available on the system that runs the app. For more information on version binding at compile time, see [Select the .NET Core version to use](../versions/selection.md#framework-dependent-apps-roll-forward).
+Publishing an FDE creates an app that automatically rolls-forward to the latest .NET security patch available on the system that runs the app. For more information on version binding at compile time, see [Select the .NET version to use](../versions/selection.md#framework-dependent-apps-roll-forward).
 
-For .NET Core 2.2 and earlier, you must use the following switches with the `dotnet publish` command to publish an FDE:
+For .NET 2.1, you must use the following switches with the `dotnet publish` command to publish an FDE:
 
 - `-r <RID>`
   This switch uses an identifier (RID) to specify the target platform. For a list of runtime identifiers, see [Runtime Identifier (RID) catalog](../rid-catalog.md).
 
 - `--self-contained false`
   This switch tells the .NET Core SDK to create an executable as an FDE.
 
+| Publish Mode                   | SDK Version | Command                                                     |
+|--------------------------------|-------------|-------------------------------------------------------------|
+| Framework-dependent executable | 3.1         | `dotnet publish -c Release -r <RID> --self-contained false`<br>`dotnet publish -c Release` |
+|                                | 5.0         | `dotnet publish -c Release -r <RID> --self-contained false`<br>`dotnet publish -c Release` |
+
 Whenever you use the `-r` switch, the output folder path changes to: `./bin/<BUILD-CONFIGURATION>/<TFM>/<RID>/publish/`
 
-If you use the [example app](#sample-app), run `dotnet publish -f netcoreapp2.2 -r win10-x64 --self-contained false`. This command creates the following executable: `./bin/Debug/netcoreapp2.2/win10-x64/publish/apptest1.exe`
+If you use the [example app](#sample-app), run `dotnet publish -f net5.0-r win10-x64 --self-contained false`. This command creates the following executable: `./bin/Debug/net5.0/win10-x64/publish/apptest1.exe`
```

Comment:
This is already merged, but note that there's a space missing after `net5.0` in the publish command.